### ISSUE TITLE
NaN when training on kitti dataset

### ISSUE THREAD

**Issue by FelippeRoza** [zzh8829/yolov3-tf2#165]
I'm trying to train a model for kitti dataset, but always get NaN for the losses. I've already tried to change the learning rate and checked the input format, as suggested in https://github.com/zzh8829/yolov3-tf2/issues/128, but everything looks right. 

I guess it may be related to the weird dimensions of kitti dataset, but not sure about how to solve it.
